sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestData.scala

```diff
@@ -279,6 +279,24 @@ private[sql] trait SQLTestData { self =>
     df
   }
 
+  protected lazy val skewData1: DataFrame = {
```

Comment:
let's define the data in where they are used. This file should only contains dataframes that are being used by multiple test suites.